src/main/java/io/codearte/jfairy/producer/person/locale/ge/GePassportNumberProvider.java

```diff
@@ -0,0 +1,36 @@
+package io.codearte.jfairy.producer.person.locale.ge;
+
+import io.codearte.jfairy.producer.BaseProducer;
+import io.codearte.jfairy.producer.person.PassportNumberProvider;
+
+import javax.inject.Inject;
+
+public class GePassportNumberProvider implements PassportNumberProvider {
+
+	private final BaseProducer baseProducer;
+
+	@Inject
+	public GePassportNumberProvider(BaseProducer baseProducer) {
+		this.baseProducer = baseProducer;
+	}
+
+	// District codes are not reachable at the moment.
+	private String getDistrictCode() {
+		return baseProducer.numerify("##");
+	}
+
+	// A system of Ministry of Inferior Office codes is not known at the moment.
+	private String getStateOfficeCode(String district) {
```

Comment:
What does this mean that they are not known? Is there any list of codes or random numbers are being used?

Reply:
The same as for district codes.